lib/types/measures/MeasureDefinition.ts

```diff
@@ -43,3 +43,12 @@ export interface MeasureDefinition {
    */
    valuesMappings: JSONObject;
 }
+
+
+/**
+ * mapping between measure name and type (include in devices and assets)
+ */
+export interface MeasureName {
```

Comment:
Naming: since this is representing the link maybe we could have something remembering this in the name, like `LinkedMeasure` for example, wdyt?